plugins/base/src/main/kotlin/translators/documentables/DocumentableFilteringStrategies.kt

```diff
@@ -0,0 +1,7 @@
+package org.jetbrains.dokka.base.translators.documentables
+
+import org.jetbrains.dokka.model.DFunction
+
+interface DocumentableFilteringStrategies {
+    fun shouldConstructorBeInPage(constructor: DFunction): Boolean
```

Comment:
Can you leave a KDoc that will allow plugin developers to understand what this extension method is for? Without context, it's not obvious which page this method is referring to.

Something along the lines of 

> Specifies whether [constructor] should appear in a separate constructors block/page. You may want to omit a constructor if it should not be visible to the user or is documented elsewhere (like Kotlin's primary constructor, which is rendered as part of the class signature)

Or, ideally, try to think of a better name, but I didn't manage to come up with anything short :(